h2o-py/h2o/model/PermutationFeatureImportance.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Permutation Feature Importance.
+
+:copyright: (c) 2020 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+
+from h2o.frame import H2OFrame
+from h2o.expr import ExprNode
+
+
+def permutation_varimp(validation_frame, model, use_pandas=True):
```

Comment:
Why validation frame? I suggest pass model as first parameter. If the frame will be None, you can use training frame from the model. 

```suggestion
def permutation_varimp(model, frame=None, use_pandas=True):
```